### PR TITLE
Add minor fixes to the translation guide

### PR DESCRIPTION
**Pull request by patiencedaur**

### CODE REVIEW

File: doc/contributing/docs/localization/guide.rst [hunk 1]
```diff
@@ -84,10 +81,13 @@ Let the resulting text sound as though it was originally written in Russian.
 Topic and focus
 ~~~~~~~~~~~~~~~
 Avoid English word order.
-The Russian speech is structured with topic and focus (тема и рема).
+
+The Russian speech is structured with topic and focus
+(:ref:`тема и рема <https://ru.wikipedia.org/wiki/Актуальное_членение_предложения>`_).
```
Comment (ValeriaKhokha) [inline doc/contributing/docs/localization/guide.rst:86]: We user :ref: for the links to the pages within Tarantool documentation.
Wiki is an external source, so an ordinary link will do just fine.
`тема и рема <https://ru.wikipedia.org/wiki/Актуальное_членение_предложения>`_

Also, not sure if "Актуальное_членение_предложения" will work here as a part of the link, but let's try.

**Comment by NickVolynkin** [comment]
Too much commits. Make a rebase on the latest branch:
```
git fetch
git rebase origin/latest
git push -f
```

File: doc/contributing/docs/localization/guide.rst [hunk 2]
```diff
@@ -66,12 +66,9 @@ Term choice examples
         *   -   state machine
             -   машина состояний (конечный автомат, state machine)
             -   машина состояний
-        *   -   “write-ahead log (WAL)” at first entry, “WAL” later
```
Comment (NickVolynkin) [inline doc/contributing/docs/localization/guide.rst:69]: why?

Comment (patiencedaur) [reply doc/contributing/docs/localization/guide.rst:69]: The examples are plain terms without descriptions. The previous version includes descriptions, which might create confusion and is also inconsistent with the previous example. Same for the punctuation.

File: doc/contributing/docs/localization/guide.rst [hunk 3]
```diff
@@ -141,13 +142,12 @@ Do all the pronouns point to the exact nouns you want them to?
...
+In the example, it is not quite clear what «к ней» means---
+to the record or to the configuration. For more on this issue, check out
 `the writers' reference at «Ошибкариум» <https://lapsus.timuroki.ink/pest/wanderer/>`_.
 
-Be critical
-~~~~~~~~~~~
+Be critical towards your text
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Tidy up your translation until it feels just right.
```
[the remark this answers is not in the document]
Comment (patiencedaur) [reply doc/contributing/docs/localization/guide.rst:152]: This is an admonition against being inattentive or uncaring about the text. I could rephrase it as follows: "Don't forget to proofread. Check your translation at least twice." This is not the same thought, but it's along similar lines. Does that sound okay to you?

File: doc/contributing/docs/localization/guide.rst (hunk 1 above)
Comment (NickVolynkin) [reply doc/contributing/docs/localization/guide.rst]: That's true. The `:doc:` and `:ref:` syntax is for referencing objects within the documentation project. These objects can be documents (pages), anchors, terms, function definitions, and so on. External links use the following syntax:
```
text text `link label <link URL>`__
```

**Comment by patiencedaur** [comment]
> [Too many commits](https://github.com/tarantool/doc/pull/2267/commits). Make a rebase on the latest branch:
> 
> ```
> git fetch
> git rebase origin/latest
> git push -f
> ```

I did `git pull --rebase origin gh-2263-minor-fixes` and `git push` instead of force-pushing.
Next step is learning about interactive rebase...

**Pull request merged**